src/components/PastTours/PastTourDetails.tsx

```diff
@@ -1,9 +1,94 @@
 import React from 'react'
+import './PastTours.css'
+import question from '../../assets/question-sign.svg'
+import route from '../../assets/travel.svg'
+import lightbulb from '../../assets/light-bulb (1).svg'
 
-export default function PastTourDetails() {
+interface tourPlan {
+  id: number;
+  tourId: number;
+  hazardWeather: string;
+  hazardAvalanche: string;
+  hazardSummary: string;
+  routePreview: string;
+  routeAlternative: string;
+  emergencyPlan: string;
+  rideObservations: string;
+  debriefConditions: string;
+  debriefDecisions: string;
+  debriefPlan: string;
+}
```

Comment:
Don't need to do this now but we can maybe have the form and PastTourDetails share an interface

Reply:
Yeah I was thinking that too! Definitely a ton of refactoring to be done